### ISSUE TITLE
Color contrast issues on /home

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75593]
### Quick summary

Three elements on /home don't have enough color contrast to pass an accessibility audit. Either the color needs to be adjusted, or the font size needs to be increased.

The elements are:
`<button type="button" class="button site-setup-list__task-action task__action is-jetpack-branded is-primary">Download mobile app</button>`
`<span class="site-setup-list__nav-item-email">User Email Address</span>`
and
`<a href="https://apps.wordpress.com/get/?campaign=calypso-customer-home" class="button go-mobile__link-button is-link">Get the Jetpack app</a>`

### Steps to reproduce

1. On /home.
2. Either manually run the elements referenced above through a color contrast checker or use a browser extension like axe DevTools Chrome extension.

### What you expected to happen

All elements should have a contrast ratio of 4.5:1 at normal text size.

### What actually happened

Some elements do not pass a color contrast check.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by taipeicoder** [comment]
Closing via #77191.
